diff --git a/vendor/magento/module-catalog/Model/Product.php b/vendor/magento/module-catalog/Model/Product.php
index 0191c02..73349d7 100644
--- a/vendor/magento/module-catalog/Model/Product.php
+++ b/vendor/magento/module-catalog/Model/Product.php
@@ -881,6 +881,36 @@ class Product extends \Magento\Catalog\Model\AbstractModel implements
             $this->setRequiredOptions(false);
         }

+        if ($this->getTypeId() == 'configurable') {
+            $writer = new \Zend_Log_Writer_Stream(BP . '/var/log/product-option.log');
+            $logger = new \Zend_Log();
+            $logger->addWriter($writer);
+            $om = \Magento\Framework\App\ObjectManager::getInstance();
+            $resource = $om->get(\Magento\Framework\App\ResourceConnection::class);
+            $connection = $resource->getConnection();
+            $query = $connection->select()->from(
+                'catalog_product_option', 'count(option_id) as count_options',
+            )->where(
+                'product_id = ?', $this->getId()
+            );
+             // Print log
+            if (empty($this->getOptions()) && !empty($connection->fetchOne($query))) {
+                $logger->info('----------Start beforeSave!----------'); // Print string type data
+                $logger->info('product Id: '.$this->getId()); // Print product Id
+                $logger->info('count options from DB: '. $connection->fetchOne($query)); // count options from DB
+                $logger->info('count options from object: '. (empty($this->getOptions()) ? 0 : count($this->getOptions()))); // count options from Object
+                try {
+                    $e = new \Exception();
+                    $logger->info($e->getTraceAsString());
+                }   catch (\Exception $e) {
+
+                    // Do nothing
+                }
+                $logger->info('----------End beforeSave!----------'); // Print string type data
+            }
+
+        }
+
         $this->getTypeInstance()->beforeSave($this);

         $hasOptions = $this->getData('has_options') === "1" && $this->isProductHasOptions();
